docs/iris_archive.md

```diff
@@ -28,6 +28,39 @@ To easily browse the IRIS Archive with a graphical interface, Azure Storage Expl
 
 For each dataset Azure Storage Explorer caches 5000 files, and only shows these by default. To load additional files the `Load more` button must be pressed.
 
+
+### Tag filtering
+
+All of the blobs inside the IRIS Archive have 3 tags that can be used for filtering:
```

Comment:
This line feels a bit clunky, maybe something more concise like: 'IRIS Archive blobs have 3 tags to filter by:'